tools/mm_eval/inception_metrics/dataset.py

```diff
@@ -117,6 +118,7 @@ def __post_init__(self):
                     num_samples_from_source = total_frame_num - self.seq_length + 1
                     for start_frame in range(0, num_samples_from_source):
                         self.start_frames.append((video_path, start_frame, num_samples_from_source))
+                    container.streams.video[0].close()
```

Comment:
Why not use `close_video(container)` here, is it because you don't need(strict=False) or because of other reasons?

Reply:
because not use load_video here, and strict=False is required here

Reply:
have changed to use `load_video` and `close_video`